src/assets/icon/IconAdd.js

```diff
@@ -1,15 +1,36 @@
 import * as React from 'react';
-import Svg, {Path} from 'react-native-svg';
+import PropTypes from 'prop-types';
+import Svg, {G, Path, Defs, ClipPath} from 'react-native-svg';
 
-const IconAdd = (props) => (
-  <Svg width={20} height={20} fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
-    <Path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M10 20C4.477 20 0 15.523 0 10S4.477 0 10 0s10 4.477 10 10-4.477 10-10 10Zm0-18a8 8 0 1 0 0 16 8 8 0 0 0 0-16Zm0 2.757a1 1 0 0 1 1 1V9h3.243a1 1 0 1 1 0 2H11v3.243a1 1 0 0 1-2 0V11H5.756a1 1 0 1 1 0-2H9V5.757a1 1 0 0 1 1-1Z"
-      fill="#000"
-    />
-  </Svg>
-);
+function IconAdd(props) {
+  return (
+    <Svg
+      viewBox="0 0 24 24"
+      width={props?.width ?? 20}
+      height={props?.height ?? 20}
+      fill={props?.fill}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining (`props?.width`) and nullish coalescing operator (`?? 20`) is a good practice to provide default values for the `width`, `height`, and `fill` properties. This ensures that even if these properties are not provided, the component will still have reasonable defaults.

```diff
- <Svg width={20} height={20} fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+ <Svg
+   viewBox="0 0 24 24"
+   width={props?.width ?? 20}
+   height={props?.height ?? 20}
+   fill={props?.fill}
+   xmlns="http://www.w3.org/2000/svg"
+   {...props}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->